src/publicodes-state/hooks/useRule/hooks/useSetValue.ts

```diff
@@ -52,7 +52,16 @@ export default function useSetValue({
             dottedName: mosaicChildDottedName,
             situation,
           })
-          if (!isMissing) return accumulator
+          const isAucunBeingSelected =
```

Comment:
Essayons de limiter le franglais si ça te va 🥲 

```suggestion
          const isNoneOptionBeingSelected =
```